cvat/requirements/base.txt

```diff
@@ -45,6 +45,8 @@ tensorflow==2.4.1 # Optional requirement of Datumaro
 patool==1.12
 diskcache==5.0.2
 open3d==0.11.2
+boto3==1.16.26
```

Comment:
Please make sure these packages and all dependencies have acceptable license

Reply:
Probably need to update boto3 version.